.github/workflows/check-upstream.main.kts

```diff
@@ -36,7 +37,11 @@ workflow(
                 git clone --branch master --single-branch https://bitbucket.org/snakeyaml/snakeyaml-engine.git
                 wget https://raw.githubusercontent.com/krzema12/snakeyaml-engine-kmp/${'$'}{{ github.ref }}/upstream-commit.txt
                 cd snakeyaml-engine
-                git log --oneline $(cat ../upstream-commit.txt)..master | wc -l > ../$numberOfCommitsFileName
+                UPSTREAM_COMMIT=$(cat ../upstream-commit.txt)
+                echo "See in BitBucket: https://bitbucket.org/snakeyaml/snakeyaml-engine/branches/compare/master..${'$'}UPSTREAM_COMMIT" > ../$logDiffBetweenRepos
+                echo "" >> ../$logDiffBetweenRepos
+                git log --oneline ${'$'}UPSTREAM_COMMIT..master >> ../$logDiffBetweenRepos
+                cat ../$logDiffBetweenRepos | wc -l > ../$numberOfCommitsFileName
```

Comment:
issue: Since we have a link to the diff and a blank line we number of lines will be `${number of commits} + 2`. Probably we need to calculate number of commits before adding the link to the diff

Reply:
Good spot, will fix.